packages/main/test/specs/MessageStrip.spec.js

```diff
@@ -83,30 +83,9 @@ describe("ARIA Support", () => {
 
 		resourceBundleText = await browser.executeAsync(done => {
```

Comment:
I suggested for you to delete the close button title test, because you make the same in cypress.